### DISCUSSION TITLE
I want to connect to the okx wallet. How can I configure it? I think the mapping of @ wagmi/core has this wallet device

### DISCUSSION THREAD

**Discussion by percyzt** [wagmi-dev/references#398] [Q&A]
I want to connect to the okx wallet. How can I configure it? I think the mapping of @ wagmi/core has this wallet device

**Comment by glitch-txs** [comment]
You can use the injected connector for this, taking a look at OKX wallet documentation might help.

https://wagmi.sh/react/connectors/injected

**Reply by percyzt** [reply]
Hello, I have configured the injection of OKX Wallet based on your article, but it seems that it has not taken effect. The default provider will still be automatically recognized now. For example, if I close the extension of OKX Wallet, it will call the Coinbase Wallet window. This is my configuration code. Is there a problem?
![image](https://github.com/wagmi-dev/references/assets/96770326/e96a7a96-e75f-4147-8f73-7a4cc26170ab)

**Reply by glitch-txs** [reply]
That config is specifically for Trust Wallet, you can check how to get the okx one in their docs 👇
https://www.okx.com/es-la/web3/build/docs/extension/injected-providers-set-up

**Reply by glitch-txs** [reply]
Btw wagmi only supports EVM compatible chains, if you want support for other chains you'll need to use their SDK and handle it separately I think. I haven't played with OKX wallet yet tbh so I'm not sure...

**Reply by dellwatson** [reply]
@glitch-txs  doesnt work either
